pkg/lib/cockpit-components-table.jsx

```diff
@@ -41,7 +41,8 @@ import './listing.less';
  * - columns: { title: string, header: boolean, sortable: boolean }[] or string[]
  * - rows: {
  *      columns: (React.Node or string)[],
- *      extraClasses: string[]
+ *      extraClasses: string[],
+ *      rowKey: ({ rowData: IRowData, rowIndex: Number }) => string | if not specified the rowIndex will be used as each row's key,
```

Comment:
I suppose "IRowData" is a typo? Would be good to give an example of its structure, as by our IRC conversation this is an utterly strange object.